raiden/ui/cli.py

```diff
@@ -73,6 +73,16 @@ def toggle_trace_profiler(raiden):
         raiden.profiler.start()
 
 
+def wait_for_sync(blockchain_service):
+    if blockchain_service.is_synced():
```

Comment:
Not a requirement, but if we actually do use the oracle idea, then we can have the `is_synced` (or another secondary) function return the number of blocks till we are synced.

This way we can also be updating the `print()` message below so that the user can be seeing how many blocks until we are synced and get an update every `gevent.sleep()` seconds.